modules/ROOT/pages/upgrade.adoc

```diff
@@ -32,6 +32,8 @@ Before migrating, ensure that you have performed and met the following prerequis
 
 * Ensure that your environment meets all of the system and network requirements described in xref:prereq-workflow.adoc[Workflow: Install and Verify Prerequisites for Physical Servers]
 
+* Ensure that the necessary kernel modules are enabled as described in xref:private-cloud::prereq-software#kernel-modules[Software Requirements for Anypoint Private Cloud, role=external, window=_blank]. 
```

Comment:
You are missing the .adoc extension for the xref.
xref:private-cloud::prereq-software.adoc#kernel-modules[Software Requirements for Anypoint Private Cloud]. 
I would also remove role=external, window=_blank

Reply:
Got these - thank you!